modules/optimeraRtdProvider.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * This module adds optimera provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ *
+ * The module will fetch targeting values from the Optimera server
+ * and apply the tageting to each ad request. These values are created
+ * from the Optimera Mesaurement script which is installed on the
+ * Publisher's site.
+ *
+ * @module modules/optimeraRtdProvider
+ * @requires module:modules/realTimeData
+ */
+
+/**
+ * @typedef {Object} ModuleParams
+ * @property {string} clientID
+ * @property {string} optimeraKeyName
+ * @property {string} device
+ */
+
+import * as utils from '../src/utils.js';
+import { submodule } from '../src/hook.js';
+import { ajaxBuilder } from '../src/ajax.js';
+
+/** @type {ModuleParams} */
+let _moduleParams = {};
+
+/**
+ * Default Optimera Key Name
+ * This can default to hb_deal_optimera for publishers
+ * who used the previous Optimera Bidder Adapter.
+ * @type {string} */
+export let optimeraKeyName = 'hb_deal_optimera';
```

Comment:
why is the integration example different?

Reply:
I set this to a default key name value for publishers porting from the current Optimera bidder adapter. New publishers will be able to set this key name to whichever key they import the Optimera values to. The example shows the key name is configurable and will override this default.